src/bindings/python/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Should probably be the LGPL, not the GPL?

Reply:
Updated to use LGPL